actionpack/lib/abstract_controller/caching/fragments.rb

```diff
@@ -135,6 +135,7 @@ def expire_fragment(key, options = nil)
       end
 
       def instrument_fragment_cache(name, key) # :nodoc:
+        return yield unless enable_fragment_cache_logging
```

Comment:
This isn't right. We don't want to disable instrumentation of fragment caches. We just don't want to show the logs, that's why I think we should look into using stuff like `Rails.logger.silence` or `cache_store.logger.mute` if the config is set.